standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/client/TestAddPartitionsFromPartSpec.java

```diff
@@ -265,18 +265,17 @@ public void testAddPartitionSpecUpperCaseDBAndTableNameInOnePart() throws Except
 
     String rootPath = tableLocation + "/addpartspectest/";
     PartitionSpecProxy partitionSpec = buildPartitionSpec(DB_NAME, tableName, rootPath, partitions);
-    client.add_partitions_pspec(partitionSpec);
+    try {
+      client.add_partitions_pspec(partitionSpec);
+      Assert.fail("MetaException should have been thrown.");
+    } catch (MetaException e) {
+      // Expected exception
+      System.out.println(e);
+    }
 
-    Partition part = client.getPartition(DB_NAME, tableName, "year=2013");
-    Assert.assertNotNull(part);
-    Assert.assertEquals(tableName, part.getTableName());
-    Assert.assertEquals(DB_NAME, part.getDbName());
-    Assert.assertEquals(tableLocation + "/year=2013", part.getSd().getLocation());
-    part = client.getPartition(DB_NAME, tableName, "year=2014");
-    Assert.assertNotNull(part);
-    Assert.assertEquals(tableName, part.getTableName());
-    Assert.assertEquals(DB_NAME, part.getDbName());
-    Assert.assertEquals(tableLocation + "/year=2014", part.getSd().getLocation());
+    List<String> partitionNames = client.listPartitionNames(DB_NAME, tableName, MAX);
```

Comment:
I think we should still keep the above lines L270-279. I think it makes sense to assert both of results of getPartitions() and listPartitions() apis

Reply:
After we reuse `add_partitions_core` function for `add_partitions_pspec` api, it will fail for the unmatched UpperCase db and table name, so actually we will not add these partitions in this test, so I remove those test lines.

Reply:
@wecharyu - So before your change, are the partitions getting added to HMS?

Reply:
Yes, `add_partitions_pspec` api does not check the name.

Reply:
Addressed! Thanks for your kind review~